Automattic/jetpack
# Jetpack Connection: Remove remaining in-place iFrame connection logic.

#### Changes proposed in this Pull Request:

Removes the in-place connection from the full-screen plugin connection, as well as most of the underlying logic that allowed in-place connections to happen, including packages. 

#### Jetpack product discussion


Lots of it. Start here p9dueE-4TL-p2, p1HpG7-bJT-p2 

There's a recent one about how the Google sign-in via in-place will stop working at the end of the month. p7H4VZ-3uJ-p2.

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






So, we'll want to make sure all the connection flows work. I tested in the following, in a browser that supported 3rd party cookies: 
- Full-screen connection prompt. Only displays immediately after activating the plugin on the plugins page. 
- Connection banner at the top of the main wp-admin dashboard. 
- /wp-admin/admin.php?page=jetpack#/setup
- When connected, test the reconnect flow `/wp-admin/admin.php?page=jetpack#/reconnect`
- When site connected, test the `/wp-admin/admin.php?page=jetpack#/connect-user` flow
- Anything else you can think of.

Since this removes a lot of in-place logic from the connection packages, we'll need to test in other plugins as well.
